prowler/providers/aws/services/dlm/dlm_ebs_snapshot_lifecycle_policy_exists/dlm_ebs_snapshot_lifecycle_policy_exists.py

```diff
@@ -0,0 +1,18 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.dlm.dlm_client import dlm_client
+
+
+class dlm_ebs_snapshot_lifecycle_policy_exists(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.status = "FAIL"
+        report.status_extended = "No EBS snapshot lifecycle policies found."
+        report.resource_id = dlm_client.audited_account
+        report.resource_arn = dlm_client.audited_account_arn
+        report.region = "global"
```

Comment:
This value is not valid, you should use the resource region or the client region instead.

Reply:
The rule is verifying is there is lifecycle policies defined in all regions, if one of the regions have these kind of policies defined the rule is return PASS result, in that case makes no sense to include one region in the report. What maybe can make sense is to divide the results per region, instead ok making a global verification. Let me check if it's OK to go that way. In the case the rule requirement doesn't allow me to do that, what should I assign to region in global verifications?

Reply:
In this case, having just one global finding, you must use `dlm_client.region`. `global` is not a valid region within the AWS provider.